.github/workflows/gradle.yml

```diff
@@ -0,0 +1,25 @@
+name: Gridmenu Action
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Lint with gradlew
+        run: ./gradlew lintDebug -S --warning-mode all
```

Comment:
Hi @Aldiwildan77 
Thank you very much for contributing this automation.
But I wonder, what is the guideline used for the linter?